### PR TITLE
Release v0.14.0

### PR DESCRIPTION
**Pull request by alesforz**
This PR creates release `v0.14.0` of cometbft-db.

This release reinstates boltdb and cleveldb as deprecated backend types, but marks them as deprecated.

### CODE REVIEW

File: .changelog/v0.14.0/summary.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+*Aug 9, 2024*
+
+This release reinstates boltdb and cleveldb as deprecated backend types.
+Please not that we discourage using them, as we plan to drop them in a future release.
```
Comment (andynog) [inline .changelog/v0.14.0/summary.md:4]:
```suggestion
Please note that we discourage the use of them, as we plan to discontinue support in a future release.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,17 +1,24 @@
 # CHANGELOG
 
+## v0.14.0
+
+*Aug 9, 2024*
+
+This release reinstates boltdb and cleveldb as deprecated backend types.
+Please not that we discourage using them, as we plan to drop them in a future release.
```
Comment (andynog) [inline CHANGELOG.md:8]:
```suggestion
Please note that we discourage the use of them, as we plan to discontinue support in a future release.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,17 +1,24 @@
...
+  ([\#177](https://github.com/cometbft/cometbft-db/pull/177))
+
 ## v0.13.0
 
 *Aug 2, 2024*
 
 This release:
-- changes the contract of the Iterator Key() and Value() APIs: the caller
-is now responsible for creating a copy of their returned value if they want to modify it.
-- bumps the Go version to 1.22.5 (from 1.22). Go 1.22.5 does not introduce new language
```
Comment (andynog) [inline CHANGELOG.md:10]: we did bump go to 1.22.5 right ? if so, we need to keep this here

**Review by andynog — APPROVED** [review]
@alesforz looks good to me, minor comments/suggestions

File: CHANGELOG.md (hunk 3 above)
Comment (alesforz) [reply CHANGELOG.md:10]: yes we did

**Pull request merged**